utils/java-dynamic-memory-opts

```diff
@@ -21,4 +21,8 @@ if [ -z "$MEM_TOTAL_KB" ]; then
 fi
 MEM_JAVA_KB=$(($MEM_TOTAL_KB * $MEM_JAVA_PERCENT / 100))
 
-echo "-Xmx${MEM_JAVA_KB}k"
+if [ -z "MEM_JAVA_XMX_EQ_XMS" ]; then
```

Comment:
not that much opinions on the PR otherwise, but you might be missing a `$`, if you are meaning to refer to a variable here.

Reply:
Of course, good catch, corrected.